app/controllers/schedules_controller.rb

```diff
@@ -3,7 +3,7 @@ def show
     @sessions = SessionQuery.new(
       relation: current_user.sessions.where(conference: current_conference),
       params: filter_params
-    ).call.includes(:attendees, :location, :speakers, :tags).order(:starts_at)
+    ).call.includes(:location, :tags, speakers: [profile: :image_attachment]).order(:starts_at)
```

Comment:
This comment applies to the 2 actions amended in `SessionsController` below as well. The `#attendees` association isn't utilised in any subsequent views, so it's an unnecessary query. Images are queried per-speaker in `sessions/show` and `sessions/_card` which are N+1s.